telemetry_examples/src/main/java/com/newrelic/telemetry/examples/TelemetryClientExample.java

```diff
@@ -32,18 +31,31 @@
   public static void main(String[] args) throws Exception {
     String insightsInsertKey = args[0];
 
+    MetricBatchSenderFactory metricFactory =
+        MetricBatchSenderFactory.fromHttpImplementation(OkHttpPoster::new);
+
     MetricBatchSender metricBatchSender =
-        SimpleMetricBatchSender.builder(insightsInsertKey, Duration.of(10, ChronoUnit.SECONDS))
-            .build();
-    SpanBatchSender spanBatchSender = SimpleSpanBatchSender.builder(insightsInsertKey).build();
-
-    // todo: make a simple version of this.
-    EventBatchSender eventBatchSender =
-        EventBatchSender.builder()
-            .apiKey(insightsInsertKey)
-            .httpPoster(new OkHttpPoster(Duration.ofSeconds(1)))
-            .enableAuditLogging()
-            .build();
+        MetricBatchSender.create(
+            metricFactory
+                .configureWith(insightsInsertKey, Duration.of(10, ChronoUnit.SECONDS))
+                .build());
+
+    SpanBatchSenderFactory spanFactory =
+        SpanBatchSenderFactory.fromHttpImplementation(OkHttpPoster::new);
+    SpanBatchSender spanBatchSender =
+        SpanBatchSender.create(spanFactory.configureWith(insightsInsertKey).build());
+
+    // a fully customized example
```

Comment:
I don't know if both of these need to be kept in here. Might be nice to eventually break these examples out into more specific use-cases that demonstrate the simpler usage, or the more complex options when needed.  For this PR, let's keep just the simple example and get rid of the fully-customized version.